deploy/template.yaml

```diff
@@ -1278,8 +1419,14 @@ Resources:
         IPVCorePrivateAPI:
           Type: Api
           Properties:
-            RestApiId:
-              Ref: IPVCorePrivateAPI
+            RestApiId: !Ref IPVCorePrivateAPI
+            Path: /journey/{journeyStep+}
+            Method: POST
+        IPVCoreInternalTestingApi:
```

Comment:
Should this one still be here?

Reply:
Yep - this approach works when attaching to a step function, rather than a lambda function. Thanks CloudFormation.

Reply:
of course